<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>449</width>
    <height>438</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true">QWidget#centralwidget{
	background-color: qradialgradient(spread:repeat, cx:0.5, cy:0.5, radius:0.077, fx:0.5, fy:0.5, stop:0 rgba(91, 0, 128, 255), stop:0.497326 rgba(0, 0, 0, 255), stop:1 rgba(78, 0, 117, 255));
}

QLabel{
	font-size: 40px;
	color: #fff;
	font-weight: bold;
}

QPushButton{
	font-size: 20px;
	font-weight: bold;
	color: white;
	min-height: 55px;
	border-radius: 15px;
	background-color: rgb(99, 0, 149);
}</string>
   </property>
   <widget class="QLabel" name="pergunta">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>421</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>?????</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QFrame" name="frame_yes">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>220</y>
      <width>141</width>
      <height>111</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::NoFrame</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="button_yes">
       <property name="text">
        <string>yes</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QFrame" name="frame_no">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>220</y>
      <width>141</width>
      <height>111</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::NoFrame</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QPushButton" name="button_no">
       <property name="text">
        <string>no</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="coracao">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>140</y>
      <width>251</width>
      <height>231</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="resource.qrc">:/images/coracao.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="resource.qrc"/>
 </resources>
 <connections/>
</ui>
